services/blockchain-connector/shared/sdk/cache.js

```diff
@@ -26,15 +26,9 @@ const cacheTrxIDToBlockIDSchema = require('../database/schema/transactions');
 
 const config = require('../../config');
 
-const getBlocksCache = () => getTableInstance(
-	cacheBlockSchema.tableName,
-	cacheBlockSchema,
-);
-
-const getTrxIDtoBlockIDCache = () => getTableInstance(
-	cacheTrxIDToBlockIDSchema.tableName,
-	cacheTrxIDToBlockIDSchema,
-);
+const MYSQL_ENDPOINT = config.endpoints.mysql;
+const getBlocksCache = () => getTableInstance(cacheBlockSchema, MYSQL_ENDPOINT);
+const getTrxIDtoBlockIDCache = () => getTableInstance(cacheTrxIDToBlockIDSchema, MYSQL_ENDPOINT);
```

Comment:
Sorry for the confusion. This was correct before. The sqlite implementation doesn't accept the mysql connection string 🤦 
```suggestion
const getBlocksCache = () => getTableInstance(cacheBlockSchema);
const getTrxIDtoBlockIDCache = () => getTableInstance(cacheTrxIDToBlockIDSchema);
```